timesheet_validation_status_enterprise/tests/test_timesheet_validation.py

```diff
@@ -0,0 +1,28 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo.tests import common
+
+
+class TestTimesheetValidation(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.project = cls.env['project.project'].create({
+            'name': 'My Project',
+        })
+        cls.task = cls.env['project.task'].create({
+            'name': 'My Task',
+            'project_id': cls.project.id,
+        })
+        cls.line_1 = cls.env['account.analytic.line'].create({
+            'name': '/',
+            'project_id': cls.project.id,
+            'task_id': cls.task.id,
+        })
+
+    def test_field_validated_is_based_on_field_validated_timesheet(self):
+        assert not self.line_1.validated
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/numigi/odoo-timesheet-addons/pullRequest?prid=4667819)